smac/tae/execute_ta_run.py

```diff
@@ -124,9 +129,12 @@ def start(self, config, instance,
                                                           seed=seed,
                                                           instance_specific=instance_specific)
 
-        if self.stats.ta_runs == 0 and status in [StatusType.CRASHED, StatusType.ABORT]:
+        if self.stats.ta_runs == 0 and status == StatusType.CRASHED:
             self.logger.critical("First run crashed -- Abort")
```

Comment:
This should now rather be the message of the exception than a call to the logger.